src/Tests/TestComponentCSharp/TestComponentCSharp.idl

```diff
@@ -241,6 +241,14 @@ namespace TestComponentCSharp
         Windows.Foundation.Collections.IVectorView<IProperties1> GetInterfaceVector();
         [noexcept] Windows.Foundation.Collections.IVectorView<Class> GetClassVector();
 
+        Windows.Foundation.Collections.IVectorView<Class> GetClassVectorSubset(Windows.Foundation.Collections.IVectorView<Class> classVector, Int32 startIndex);
```

Comment:
shouldn't this all go into TestComponent?

Reply:
Some of this can go in TestComponent, but not all such as the Blittable ones which are only defined here.  I will look at moving the others.

Reply:
Actually realized testwinrt has them too, so will look at moving them all.